cpp/ql/test/query-tests/Security/CWE/notice.txt

```diff
@@ -0,0 +1,2 @@
+
+Some of the the files in these tests contain source code copied or derived from the public domain "Juliet Test Suite for C/C++" (provided by NIST / SAMATE Team at https://samate.nist.gov/SARD/testsuite.php).  Such tests are typically in subdirectories named "SAMATE".
```

Comment:
Can this file be named `README` instead? Or some other file name that GitHub will consider to be a readme and therefore display when listing the directory.

Reply:
Good idea (there's no need for it to be `notice.txt`, and having it more visible on GitHub seems like a plus).

Done.